lib/mayaUsd/utils/util.cpp

```diff
@@ -185,6 +185,17 @@ std::string UsdMayaUtil::GetMayaNodeName(const MObject& mayaNode)
     return nodeName.asChar();
 }
 
+MString UsdMayaUtil::GetUniqueNameOfDAGNode(const MObject& node)
+{
+    TF_VERIFY(!node.isNull());
+    TF_VERIFY(node.hasFn(MFn::kDagNode));
```

Comment:
I'll just mention on this one instance here that `TF_VERIFY` won't prevent execution from continuing through the rest of this function, so you might want to do something like this:

```C++
if (!TF_VERIFY(!node.isNull() && node.hasFn(MFn::kDagNode)) {
    return MString();
}
...
```